.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.1
```

Comment:
Rails 5.2 now creates the ruby-version file by default, so I figured I'd include it. See: https://github.com/rails/rails/pull/30016. I'm happy to remove it if you'd prefer, though!

Reply:
It seems as though rbenv can read from the .ruby-version file but not from the gemfile. With the .ruby-version I am able to `bundle` without setting a project ruby version using rbenv, but my global ruby version is used by default otherwise.